### DISCUSSION TITLE
I can't filter with `display_name` for the `DeviceType`

### DISCUSSION THREAD

**Discussion by rimoi** [netbox-community/netbox#5477] [Q&A] [answered]
### Steps to Reproduce
- I just want to make this request : 
```sql
SELECT * FROM device_type WHERE display_name LIKE '%brocad%'
```
To do this, I call this url in `GET` :
`https://website-dcim/api/dcim/device-types/?display_name_ic=brocad`

### Expected Behavior
I just want filtered results.

### Observed Behavior
But it doesn't filter, it returns all the DeviceType data I have in my database.

Thanks

**Comment by jeremystretch** [comment, marked as answer]
`display_name` isn't a field which exists in the database: It's a concatenation of the device type's manufacturer name and model. Because `manufacturer` is a related field, it can only be filtered using an exact match; e.g. `?manufacturer=brocade` or `?manufacturer_id=123`. You can, however, filter against the model number using a partial match.

The complete list of available filters for each API endpoint is listed in the API documentation at `/api/docs/`.

**Comment by rimoi** [comment]
thanks for your answer
